webxrlayers-1.bs

```diff
@@ -0,0 +1,730 @@
+
+<div class="example">
+The following code requests layers as an optional feature.
+
+<pre highlight="js">
+navigator.xr.requestSession('immersive-vr', {
+    optionalFeatures: ['layers']
+  }
+</pre>
+</div>
+
+Layers are only supported for XRSessions created with XRSessionMode of {{XRSessionMode/"immersive-vr"}} 
+or {{XRSessionMode/"immersive-ar"}}. {{XRSessionMode/"inline"}} sessions must not support layers.
+
+Layer types {#xrlayertypes}
+===========
+
+Mono and stereo layers {#monovsstereo}
+----------------------
+A stereo layer must supply a different view to render to for each eye. Stereo layers should be refreshed 
+close to the device's native frame rate.
+
+A mono layers must supply a single view which is shown to each eye. There is no requirement to refresh mono 
+layers repeatedly.
+
+The [=XR Compositor=] must ensure that layers are presented correctly in stereo to the observer.
+
+XRLayer {#xrlayertype}
+-------
+Every layer except XRWebGLLayer is derived from XRLayer.
+XRLayer defines a set of common attributes and behaviors acress all layer types.
+
+<pre class="idl">
+interface XRLayer {
+  readonly attribute unsigned long pixelWidth;
+  readonly attribute unsigned long pixelHeight;
+
+  attribute boolean blendTextureSourceAlpha;
+  attribute boolean chromaticAberrationCorrection;
+
+  void destroy();
+};
+</pre>
+
+The <dfn attribute for="XRLayer">pixelWidth</dfn> and <dfn attribute for="XRLayer">pixelHeight</dfn> attributes 
+return the width and height of the GL attachments, respectively. If there are no attachments, they will return 0.
+
+The <dfn attribute for="XRLayer">blendTextureSourceAlpha</dfn> attribute enables the layer’s texture alpha channel.
+
+The <dfn attribute for="XRLayer">chromaticAberrationCorrection</dfn> attribute enables optical chromatic 
+aberration correction for the layer when not done by default.
+
+{{destroy()}} will delete the underlying attachments. If there are no attachments, this function does nothing.
+
+Each {{XRLayer}} has a <dfn for="XRLayer">context</dfn> object which is an instance 
+of either a {{WebGLRenderingContext}} or a {{WebGL2RenderingContext}}.
+
+Each {{XRLayer}} has an associated <dfn for="XRLayer">session</dfn>, which is the 
+{{XRSession}} it was created with.
+
+XRProjectionLayer {#xrprojectionlayertype}
+-----------------
+A XRProjectionLayer is a layer that fills the entire view of the observer.
+Projection layer must always run stereo and should be refreshed close to the device's native frame rate.
+
+<pre class="idl">
+interface XRProjectionLayer : XRLayer {
+  readonly attribute boolean ignoreDepthValues;
+};
+</pre>
+
+The <dfn attribute for="XRProjectionLayer">ignoreDepthValues</dfn> attribute, if <code>true</code>, indicates the 
+[=XR Compositor=] MUST NOT make use of values in the depth buffer attachment when rendering. When the attribute 
+is <code>false</code> it indicates that the content of the depth buffer attachment will be used by the 
```

Comment:
```suggestion
is <code>false</code> it indicates that the content of the depth buffer attachment can be used by the 
```

Minor change, but we shouldn't imply that all compositors will make use of the depth buffer.

Reply:
I filed an issue on this earlier today.
I was thinking that the depthstenciltexure should only be provided on platforms where the depth buffer is used by the compositior.